index.js

```diff
@@ -101,6 +102,9 @@ class I18n {
                 }
                 break;
             default:
+                if (fallback) {
```

Comment:
I'm not 100% on board with skipping the missing report. Although I can see how this would be expected behaviour - I think the side effect is unrelated, and skipping it should be explicit (an options argument?)

I may be over thinking here tho. So just consider it and decide on your own.